### PR TITLE
Cryptollar Complete Implementation

### PR DESCRIPTION
**Pull request by adamilare**
In this update, the following were implemented;

Setup React app with redux
Used CoinMarketCapp Api for data used in the app
Created components for the app
Created Redux store
Implemented a couple of routes
Implemented filtration technique for allowing users to filter displayed cryptocurrencies.
Write tests for components and other features
Deployed the app using Render
Made a video demo of the app
[Link to video](https://www.loom.com/share/fae696325ea64640a699bcbe70dc7a62)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,155 @@
...
+<a name="readme-top"></a>
+
+<div align="center">
+<img src="./src/assets/cryptollar.png" alt="cryptollar logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>CRYPTOLLAR</b></h1>
+</div>
+
+</div>
+# 📗 Table of Contents
```
Comment (rloterh) [inline README.md:10]: - [ ] [OPTIONAL]
Kindly improve upon the look of the underlined section so as the make the ReadMe appear more professional.
  
image

File: README.md [hunk 2]
```diff
@@ -1 +1,155 @@
...
+-
+
+### Usage <a name="usage">
+
+To run the project, starting the React App, execute the following command:
+
+```sh
+  npm start
+```
+
```
Comment (rloterh) [inline README.md:100]: - [ ] [OPTIONAL]
It will be a good idea to add instructions on how to run test cases for your app, in the _Getting Started_ section

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Damilare,  

Your project is complete! 🏁 Great job! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job acknowledging Nelson Sakwa on Behance for the [Original design idea](https://www.behance.net/sakwadesignstudio).
🎲  Nice presentation of this project!
🎲  Correctly deployed project!
🎲 API fetch properly implemented!
🎲 State management with Redux properly implemented!
🎲 SPA feature working as intended!
🎲 Properly implemented integration test!
🎲 Filter works great!  
🎲 Design matches given template!
🎲 Unit tests properly covered!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (adamilare) [reply README.md:10]: @rloterh Thank you for your review

**Pull request merged**